test/src/test/java/hudson/jobs/CreateItemTest.java

```diff
@@ -142,4 +149,54 @@ public void onCheckCopy(Item src, ItemGroup parent) throws Failure {
         }
     }
 
+    @Issue("JENKINS-74795")
+    @Test
+    public void testCreateItemDoesNotPopulateDefaultView() throws Exception {
+        // Create a view that only displays jobs that start with 'a-'
+        FreeStyleProject aJob = rule.createFreeStyleProject("a-freestyle-job");
+        ListView aView = new ListView("a-view");
+        aView.setIncludeRegex("a-.*");
+        rule.jenkins.addView(aView);
+        assertThat(aView.getItems(), containsInAnyOrder(aJob));
+        assertFalse(aView.isDefault()); // Not yet the default view
+
+        // Create a view that only displays jobs that start with 'b-'
+        FreeStyleProject bJob = rule.createFreeStyleProject("b-freestyle-job");
+        ListView bView = new ListView("b-view");
+        bView.setIncludeRegex("b-.*");
+        rule.jenkins.addView(bView);
+        assertThat(bView.getItems(), containsInAnyOrder(bJob));
+        assertFalse(bView.isDefault()); // Not the default view
+
+        // Make the a-view the default
+        rule.jenkins.setPrimaryView(aView);
+        assertTrue(aView.isDefault()); // Now a-view is the default view
+
+        // Use createItem API to create a new job
+        User user = User.getById("user", true);
+        rule.jenkins.setSecurityRealm(rule.createDummySecurityRealm());
+        rule.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy()
+                .grant(Jenkins.READ, Item.CREATE)
+                .everywhere()
+                .to(user.getId()));
+        String b2JobName = "b-freestyle-job-2";
+        try (JenkinsRule.WebClient wc = rule.createWebClient()) {
+            wc.login(user.getId());
+            WebRequest request = new WebRequest(wc.createCrumbedUrl("createItem?name=" + b2JobName), HttpMethod.POST);
+            request.setAdditionalHeader("Content-Type", "application/xml");
+            request.setRequestBody("<project/>");
+            wc.getPage(request);
+        }
+        FreeStyleProject b2Job = rule.jenkins.getItemByFullName(b2JobName, FreeStyleProject.class);
+        assertThat(bView.getItems(), containsInAnyOrder(bJob, b2Job));
+        assertFalse(bView.isDefault());
+
+        // Confirm new job is not visible in default view
+        assertTrue(aView.isDefault()); // a-view is still the default view
+        assertThat(aView.getItems(), containsInAnyOrder(aJob));
+
+        // Confirm new job is visible in non-default view
+        assertFalse(bView.isDefault()); // b-view is not the default view
+        assertThat(bView.getItems(), containsInAnyOrder(bJob, b2Job));
```

Comment:
Optional, these two assertions are duplicates of the assertions on line 191 and 192.  No harm to have the duplicated assertions, but it is also safe to remove them.
```suggestion
```